fluentd/lib/fluent-plugin-prometheus/in_prometheus_tail_monitor.rb

```diff
@@ -0,0 +1,108 @@
+require 'fluent/plugin/input'
+require 'fluent/plugin/in_monitor_agent'
+require 'fluent/plugin/prometheus'
+
+module Fluent::Plugin
+  class PrometheusTailMonitorInput < Fluent::Plugin::Input
+    Fluent::Plugin.register_input('prometheus_tail_monitor', self)
+    include Fluent::Plugin::PrometheusLabelParser
+
+    helpers :timer
+
+    config_param :interval, :time, default: 1
```

Comment:
Why did you change the default from 5s to 1s?

Reply:
This is hard to review because there's so much code that is copied from the original. Can you subclass the original and  override just what we need to change? That would make it easier to review and maintain.

Reply:
resolved

Reply:
resolved. have rolled back this change.